src/components/AvailableImagesTile.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {
+  Card,
+  CardTitle,
+  CardBody,
+  Button,
+  Spinner,
+} from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+
+const AvailableImageTileBase = ({ children }) => {
+  return (
+    <Card>
+      <CardTitle>Available images</CardTitle>
+      {children}
+    </Card>
+  );
+};
+
+AvailableImageTileBase.propTypes = {
+  children: PropTypes.children,
+};
+
+const AvailableImageTile = () => {
+  const { status, data } = useSelector(({ imagesReducer }) => ({
+    status: imagesReducer?.status || 'PENDING',
+    data: imagesReducer?.data || null,
+    error: imagesReducer?.error || null,
+  }));
```

Comment:
Either spread this to 2 selectors or use `shallowEqual` compare function so we don't get multiple re-renders.